vocabularies/Common.xml

```diff
@@ -114,16 +114,26 @@
         </Member>
       </EnumType>
 
-      <Term Name="Timezone" Type="Edm.String" Nullable="true" AppliesTo="Property Parameter">
+      <Term Name="Timezone" Type="Edm.String" Nullable="true" AppliesTo="Property Parameter ReturnType">
         <Annotation Term="Common.Experimental" />
-        <Annotation Term="Core.Description" String="The point in time represented by the annotated property or parameter shall be interpreted in the given time zone" />
+        <Annotation Term="Core.Description" String="The point in time represented by the annotated property, parameter or return type shall be interpreted in the given time zone" />
```

Comment:
```suggestion
        <Annotation Term="Core.Description" String="The point in time represented by the annotated property, parameter or return type must be interpreted in the given time zone" />
```
See https://datatracker.ietf.org/doc/html/rfc2119#section-1 and https://www.plainlanguage.gov/guidelines/conversational/shall-and-must/:
- "must" and "shall" mean the same
- "must" is better